extensions/wikia/EditPageLayout/js/editor/AceEditor.js

```diff
@@ -77,6 +83,17 @@ define('wikia.editpage.ace.editor', ['wikia.ace.editor', 'wikia.window'], functi
 	}
 
 	/**
+	 * Make sure that user doesn't left the page by accident
+	 */
+	function beforeUnload() {
+		$(win).bind( 'beforeunload', function() {
```

Comment:
Shouldn't have a space after `.bind(` and [should have a space](https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#functions) after the function keyword for the anonymous function per our conventions.